### DISCUSSION TITLE
[TypeError]: The NextURL configuration includes no locale

### DISCUSSION THREAD

**Discussion by Aldhanekaa** [vercel/next.js#38594] [Help] [answered]
Hello all! So I wanna ask you guys about my Nextjs website's error which currently occurs on my production server. 

So in the website I make, I implement Internationalized Routing on my website [Middleware](https://nextjs.org/docs/advanced-features/middleware). It runs well on local, no error occurs during the development stage. 

Thus I pushed the website to my server. It also runs well on my server (without a domain address), but when I point a domain ([gabutnetwork.net](gabutnetwork.net)) to my server address, however when I visited the website using that domain, out of nowhere an `500: Internal Server Error` occured, and then I went check the website with my server address, and it ran pretty well, without error occurred.

Then I tried to debug my Nextjs log, and it turns out that;
the reason why `500 Internal Server Error` occured is because of an error at `Error [TypeError]: The NextURL configuration includes no locale "en;q=0.6"` . But it's really weird, an error could be occured on the website which has domain while the same website doesn't have any error when visiting it through the server address. 

And also, while the console says `Error [TypeError]: The NextURL configuration includes no locale "en;q=0.6"`  , however the web console says `"A client-side exception has occurred"`

May I know the explanation of experience I'm facing right now? Thanks.

---

screen shots 
<img width="1438" alt="Screenshot 2022-07-13 at 20 01 32" src="https://user-images.githubusercontent.com/67791514/178740368-786c5b05-3133-45f5-b087-07c4173c1ad4.png">
<img width="1438" alt="Screenshot 2022-07-13 at 20 02 40" src="https://user-images.githubusercontent.com/67791514/178740426-b8595d4a-62a0-4ba2-8c2f-276077e45018.png">

---

info
```
"next": "^12.2.2",
"react": "18.1.0",
"next-i18next": "^11.0.0",
```

**Comment by icyJoseph** [comment, marked as answer]
Hi,

Is the site still down? or showing the issue? I can see gabutnetwork.net fine on my end.

I guess one problem could be that the `next.config.js` is missing from the deployment. IIRC, it has to follow along with the rest of the build.

**Reply by Aldhanekaa** [reply]
tbh, I'd added the next.config.js. Anyways, yeah it works just fine when I visit the site in safari, but why does the error still occur when I visit the site using Brave. hm really weird, the error still occurs even when I visit it with Brave's Incognito

**Reply by Aldhanekaa** [reply]
> tbh, I'd added the next.config.js. Anyways, yeah it works just fine when I visit the site in safari, but why does the error still occur when I visit the site using Brave. hm really weird, the error still occurs even when I visit it with Brave's Incognito

Alright solved, thanks for the explanation, I appreciate it @icyJoseph
